SteamBundleSitesExtension.js

```diff
@@ -342,7 +345,7 @@ GM_addStyle(`
 const getSessionID = () => {
     GM_xmlhttpRequest({
         method: 'GET',
-        url: 'https://store.steampowered.com/',
+        url: 'http://store.steampowered.com/',
```

Comment:
Not sure if this is necessary...
Steam might switch to https and we won't need to change code if we keep https

Reply:
For now, using https will add one more request.
First respond a 302 temporarily moved to http
and secend request http
if Valve use ssl all their website some day, we'll fix it up

Reply:
Alright, was not aware of that